CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Fixes
+- Fix endMedia to not be removed from DOM on mouseleave for `ListItem` ([#278](https://github.com/stardust-ui/react/pull/278))
```

Comment:
This is missing your github @username.

Reply:
Thanks for catching this. Will update. 

Somehow I had copied and updated the markup from the only entry which did not have it. :)
### Fixes
- Fixed fonts to support zwj gender emojis ([#215](https://github.com/stardust-ui/react/pull/215))